l10n_vn_account_invoice_number/models/account_voucher.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountVoucher(models.Model):
+    _inherit = "account.voucher"
+
+    invoice_number = fields.Integer()
+    invoice_form = fields.Char(size=10)
```

Comment:
I don't think that we should specify the size for tax_invoice_form nor tax_invoice_serie